agenta-cli/agenta/sdk/agenta_init.py

```diff
@@ -7,11 +7,20 @@
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.DEBUG)
 
-from agenta.client.client import ClientWrapper
-
-client_wrapper = ClientWrapper()
+from agenta.api.api import ClientWrapper
+from agenta.api.exceptions import APIRequestError
+
+BACKEND_URL_SUFFIX = os.environ.get("BACKEND_URL_SUFFIX", "api")
+CLIENT_API_KEY = os.environ.get("AGENTA_API_KEY")
+CLIENT_HOST = os.environ.get("AGENTA_HOST", "http://localhost")
+
+# initialize the client with the backend url and api key
+backend_url = f"{CLIENT_HOST}/{BACKEND_URL_SUFFIX}"
+client_wrapper = ClientWrapper(
```

Comment:
I don't think it makes sense to have this here. The variable api_key and host can be both provided in the init constructor in AgentaSingleton but also as env vars. Here you are only taking care of the second case. This should be inside init i think

Reply:
I set it inside init and while it worked in making requests with the client, somehow it was preventing the app from starting and the error messages were not helpful enough for me to understand why that was happening even as it was seemingly logically correct. 

When I took this approach, I stopped getting the errors and it worked.